src/Collections-Tests/DictionaryTest.class.st

```diff
@@ -617,6 +617,17 @@ DictionaryTest >> testNewFrom [
 
 ]
 
+{ #category : #'tests - new' }
+DictionaryTest >> testNewFromKeysAndValues [
+ 
+	| keys values newDict |
+	keys := self nonEmptyDict keys.
+	values := self nonEmptyDict values.
+	newDict := Dictionary newFromKeys: keys andValues: values.
```

Comment:
Instead of #newFromKeys:andValues: why not just #keys:values:? We have #with, not #newWith:. For matrix we have #rows:columns: and not #newFromRows:andColumns:. 

What do you think?

Reply:
On dictionary we have newFromPairs: and newFrom: so this is why I followed this pattern.